### PR TITLE
Book Detail V - Redirection

### PR DESCRIPTION
**Pull request by joacomiguens**
## Summary

- Create detail-book.html to contain detail-book card and its back-button.

- Update index.html to allow redirection after click.

## Screenshot

book-detail-redirection

## Trello Card

You can find it [here](https://trello.com/c/kORHkDf9/33-detalle-de-libro-vi-redirecci%C3%B3n)

### CODE REVIEW

File: detail-book.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training" />
+    <meta name="author" content="Wolox" />
+    <link rel="stylesheet" type="text/css" href="application.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="nav">
+        <div class="nav__content">
+          <img src="assets/LogoWolox.png" class="nav__content__logo" />
```
Comment (ebedoyaalzate) [inline detail-book.html]: Add alt

File: index.html [hunk 2]
```diff
@@ -18,86 +18,78 @@
...
     <div class="container">
       <div class="book-list">
-        <div class="book m-bottom-40">
-          <img src="./assets/book-cover.png" alt="" class="book__cover" />
-          <div class="book__info">
-            <h3 class="book__info__title">Título del libro</h3>
-            <h4 class="book__info__author">Autor del libro</h4>
+        <a href="detail-book.html" class="book-link">
+          <div class="book m-bottom-40">
+            <img src="./assets/book-cover.png" alt="" class="book__cover" />
```
Comment (ebedoyaalzate) [inline index.html]: Same here

File: index.html [hunk 3]
```diff
@@ -18,86 +18,78 @@
...
-        </div>
-        <div class="book m-bottom-40">
-          <img src="./assets/book-cover.png" alt="" class="book__cover" />
-          <div class="book__info">
-            <h3 class="book__info__title">Título del libro</h3>
-            <h4 class="book__info__author">Autor del libro</h4>
+        </a>
+        <a href="detail-book.html" class="book-link">
+          <div class="book m-bottom-40">
+            <img src="./assets/book-cover.png" alt="" class="book__cover" />
```
Comment (ebedoyaalzate) [inline index.html]: And the rest of img tags

File: scss/_detail.scss [hunk 4]
```diff
@@ -1,7 +1,41 @@
...
+
+.back-button {
+  align-self: flex-start;
+  margin: 0 0 40px 40px;
+
+  &__link {
+    font-size: 20px;
+    &::before {
+      bottom: -2px;
+      content: url('C:\\Users/Julieta/Desktop/Personal/Wolox/Codigo/Intro-Maquetado/jm-angular/assets/back-arrow-20.png');
```
Comment (ebedoyaalzate) [inline scss/_detail.scss]: Use a relative path or from root of the project

Comment (ebedoyaalzate) [inline scss/_detail.scss]: For back arrow you can use the img tag, because it's a functional image not just for decorative purpose

**Review by ebedoyaalzate — CHANGES_REQUESTED** [review]

File: detail-book.html (hunk 1 above)
Comment (joacomiguens) [reply detail-book.html]: Done 😄

File: index.html (hunk 2 above)
Comment (joacomiguens) [reply index.html]: Done 😄

File: index.html (hunk 3 above)
Comment (joacomiguens) [reply index.html]: Done 😄

File: scss/_detail.scss (hunk 4 above)
Comment (joacomiguens) [reply scss/_detail.scss]: Done 😄

Comment (joacomiguens) [reply scss/_detail.scss]: I use pseudo-element because Trello Card says that I have to use that 👐

**Review by ebedoyaalzate — APPROVED** [review]

File: detail-book.html [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    <meta charset="utf-8" />
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training" />
+    <meta name="author" content="Wolox" />
+    <link rel="stylesheet" type="text/css" href="application.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="nav">
+        <div class="nav__content">
```
Comment (mcallegari10) [inline detail-book.html]: I think this div is not necessary

File: detail-book.html [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+          <img src="assets/LogoWolox.png" alt="Wolox logo" class="nav__content__logo" />
+          <a href="#" class="nav__content__auth">Logout</a>
+        </div>
+      </nav>
+    </header>
+    <div class="container-detail">
+      <div class="container-detail__wrapper">
+        <div class="back-button">
```
Comment (mcallegari10) [inline detail-book.html]: Is this necessary?

File: scss/_detail.scss [hunk 7]
```diff
@@ -1,7 +1,42 @@
+.container {
+  &-detail {
+    height: 100vh;
```
Comment (mcallegari10) [inline scss/_detail.scss]: This is not necessary since we want the height to be `auto`

**Review by mcallegari10 — CHANGES_REQUESTED** [review]

File: detail-book.html (hunk 5 above)
Comment (joacomiguens) [reply detail-book.html]: Done 😄

File: detail-book.html (hunk 6 above)
Comment (joacomiguens) [reply detail-book.html]: Not at all 😰

File: scss/_detail.scss (hunk 7 above)
Comment (joacomiguens) [reply scss/_detail.scss]: Done 😄

**Review by mcallegari10 — APPROVED** [review]

**Pull request merged**